packages/avatar/src/types.ts

```diff
@@ -66,6 +66,8 @@ export interface AvatarStackProps {
   limit?: number
   showMore?: boolean
   borderColor?: ColorProperty
+  size?: number
+  hasBorder?: boolean
 }
 
```

Comment:
Replace smashing avatar defaults with this:

```tsx
declare module 'styled-components' {
  export interface SmashingAvatarDefaults
    extends Partial<{
      avatar?: Pick<AvatarProps, 'appearance' | 'size' | 'sizeLimitOneCharacter' | 'forceShowInitials'>,
      avatarStack?: Pick<AvatarStackProps, 'showMore' | 'borderColor' | 'size' | 'hasBorder'>
    }> {}
}
```

This will link to original props defined in AvatarProps/AvatarStackProps and show property comments